docs/fixers.rst

```diff
@@ -21,6 +21,12 @@ then use the ``--no-six`` flag.
 Fixers use the API defined by 2to3. For details of how this works, and how to
 implement your own fixers, see `Extending 2to3 with your own fixers, at
 python3porting.com <http://python3porting.com/fixers.html>`_.
+``python-modernize`` will try to load fixers whose full dotted-path is specified
+as a ``-f`` argument, but will fail if they are not found. Note that, by
+default, the current directory is not on the Python-Path when you run the
```

Comment:
Rather than mentionining the Python path, I'd say something like "...fixers will not be found in the current directory..."

Reply:
I think it would be incomplete not to say anything about the Python path here, but I see your point.
How about something like,
```rst
By default, fixers will not be found in the current directory; 
use ``--fixers-here`` to make ``python-modernize`` look for them there,
or see (link to relevant docs) for more info on how Python finds modules.
```

Reply:
That sounds good, though I don't know of any good explanatory docs for how imports are found. [This page](https://docs.python.org/3/reference/import.html) is much too detailed.

Reply:
https://docs.python.org/3/tutorial/modules.html and in particular https://docs.python.org/3/tutorial/modules.html#the-module-search-path and https://docs.python.org/3/tutorial/modules.html#packages

Reply:
Great, thanks!